lib/system/api/wifi.toit

```diff
@@ -7,16 +7,16 @@ import system.api.network show NetworkService NetworkServiceClient
 interface WifiService extends NetworkService:
   static UUID  /string ::= "2436edc6-4cd8-4834-8ebc-ed883990da40"
   static MAJOR /int    ::= 0
-  static MINOR /int    ::= 8
+  static MINOR /int    ::= 9
```

Comment:
If you change the method numbering (not just add new ones), we should change the MAJOR version, but for now, all WiFi implementations are coupled with the API definition, so it doesn't matter.

Reply:
Does it mean current we should not modify this version definition?